go/ai/generator_test.go

```diff
@@ -172,6 +174,8 @@ func TestValidMessage(t *testing.T) {
 		outputSchema := &ModelOutputConfig{
 			Format: string(OutputFormatJSON),
 		}
+		// FIX: validMessage compares response against a given schema, but
```

Comment:
Since `validMessage` checks if the response message matches the given schema (but no schema is provided), would it make sense to rework this and the next two test cases to not expect an error?